PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,21 @@
-This pull request `[DESCRIBE WHAT THIS PULL REQUEST IS TRYING TO ACCOMPLISH HERE]`. It `[DESCRIBE HOW IT ACCOMPLISHES THE GOAL HERE]`.
+Please add some information beneath all of these headers. Then, **delete this sentence**.
 
-See also: `[PROVIDE REFERENCES TO OTHER TICKETS, PULL REQUESTS, OR MAILING LIST THREADS HERE. OPTIONAL.]`
+### Brief technical summary of changes made
+
+
+
+### This resolves issue...
+
+- [ ] Redmine # ?
+
+- [ ] Github # ?
+
+### This PR improves LORIS by...
+
+
+
+### To test this change...
+
+- [ ] 
+
+# If this sentence shows up it means I didn't read the instructions!
```

Comment:
Sorry, I'm a little bit confused. Should we be removing both the first and last line of the template?

Reply:
At this point, yes. It's a sneaky trap